app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+  include SessionsHelper
+
+  skip_before_action :require_login
+
+  def new
+  end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user && user.password == params[:session][:password]
+      successful_session(user)
+    else
+      flash[:success] = 'Invalid email/password combination'
+      redirect_to '/users/signin'
+    end
+  end
+
+  def destroy
+    session.delete(:current_user_id)
+    flash[:success] = "You have logged out"
```

Comment:
the flash here is fine to be success because the thing did what the user intended to do.